api/python/provisioner/__init__.py

```diff
@@ -42,6 +42,7 @@
     set_swupdate_repo,
     sw_update,
     fw_update,
+    rollback_update,
```

Comment:
Should we change this to
**sw_rollback**

Reply:
@SomwanshiAnjali Agree. It sounds reasonable. Could you adjust here and in similar places?

Reply:
addressed